x-pack/platform/plugins/shared/spaces/public/management/edit_space/security_disabled_callout.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiCallOut } from '@elastic/eui';
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+
+export const SecurityDisabledCallout = () => (
+  <EuiCallOut
+    color="warning"
+    data-test-subj="securityDisabledCallout"
+    title={i18n.translate(
+      'xpack.spaces.management.spaceDetails.contentTabs.securityDisabledTitle',
+      {
+        defaultMessage: 'Security feature is disabled',
+      }
+    )}
+  >
+    {i18n.translate(
+      'xpack.spaces.management.spaceDetails.contentTabs.securityDisabledDescription',
+      {
+        defaultMessage: 'To manage permissions, enable security feature.',
```

Comment:
```suggestion
        defaultMessage: 'To manage space permissions, you must enable security features first.',
```

Wondering if it would be useful to link to https://www.elastic.co/guide/en/elasticsearch/reference/current/security-minimal-setup.html#_enable_elasticsearch_security_features in addition?